docs/astrocut/index.rst

```diff
@@ -356,10 +356,56 @@ Note that multithreading is disabled by default.
                 Total time: 7.8 sec
 
 
+
+ASDF File Cutouts
+===================
+
+The Nancy Grace Roman Space Telescope will store its data using the Advanced Scientific Data Format (ASDF). With the `asdf_cut` function, users can create cutouts of Roman mission products.
+
+Creating ASDF Cutouts
+----------------------
+
+The function `asdf_cut` performs a cutout of an ASDF file and returns the result as either a FITS file or an ASDF file. The cutout ASDF file format is described `here <file_formats.html#asdf-cutout-files>`__.
```

Comment:
Since the output can be either FITS or ASDF, maybe your second example can output an ASDF file to demonstrate if there is anything special for calling it. I assume it's just changing it to be something like roman-demo.asdf